configs/README.md

```diff
@@ -0,0 +1,25 @@
+## Configuration File
+Example configuration files are provided here. To build and run the given examples see the instructions ([here](https://github.com/NOAA-OWP/LGAR-C/blob/ajk/doc_update/INSTALL.md#build-standalone-mode)).
```

Comment:
"Example configuration files are provided in this directory"? I was half-expecting the work 'here' to point to a link but realized that it refers to this directory.

Reply:
good point, I revised it.